views/api.html

```diff
@@ -0,0 +1,198 @@
+<div class="o-layout o-layout--docs" data-o-component="o-layout">
+	<div class="o-layout__header">
+		{{> header }}
+	</div>
+
+	<div class="o-layout__sidebar"></div>
+
+	<div class="o-layout__main o-layout-typography" data-o-component="o-syntax-highlight">
+		<h1 id="#api-reference">API Reference</h1>
+		<div>
+			<h2 id="get-v3-bundle-js)">JavaScript</h2>
+			<table class="o-table o-table--horizontal-lines o-table--row-headings">
+				<tbody>
+					<tr>
+						<th scope="row">Method</th>
+						<td>
+							<code>GET</code>
+						</td>
+					</tr>
+					<tr>
+						<th scope="row">
+							Path
+						</th>
+						<td>
+							<code>/v3/bundles/js</code>
+						</td>
+					</tr>
+					<tr>
+						<th scope="row">Querystring</th>
+						<td>
+							<dl>
+								<dt>
+									<code>
+										<var>modules</var>
+									</code>
+								</dt>
+								<dd>
+									<strong>Required</strong>
+									<p>A comma-separated list of Origami components in the format <code>name@version</code>. The <code>name</code> can be any Origami component and the <code>version</code> is any <abbr title="Semantic Versioning">SemVer</abbr> <a href="https://semver.org/">compatible</a> version.</p>
+								</dd>
+								<dt>
+									<code>
+										<var>source</var>
+									</code>
+								</dt>
+								<dd>
+									<strong>Required</strong>
+									<p>The systemcode of the System that is is making ue of the Origami Build Service.</p>
+								</dd>
+								<dt>
+									<code>
+										<var>minify</var>
+									</code>
+								</dt>
+								<dd>
+									<strong>Defaults to <code>on</code></strong>
+									<p>Set to <code>on</code> to have the response be minified.</p>
+									<p>Set to <code>off</code> to have the response not be minified.</p>
+								</dd>
+								<dt>
+									<code>
+										<var>esmodules</var>
+									</code>
+								</dt>
+								<dd>
+									<strong>Defaults to <code>off</code></strong>
+									<p>Set to <code>on</code> to have the response be in <abbr title="ECMAScript Module">ESM</abbr> format.</p>
+									<p>Set to <code>off</code> to have the response be in <abbr title="Universal Module Definition">UMD</abbr> format.</p>
+								</dd>
+								<dt>
+									<code>
+										<var>ua</var>
+									</code>
+								</dt>
+								<dd>
+									<strong>Internal use only</strong>
+									<p>Set to a normalised useragent string to have the JavaScript be compiled to the latest version of JavaScript that the useragent supports.</p>
+									<p>This is used internally in the service and is exposed to simplify debugging a response and to acheive a simpler caching workflow with our CDN.</p>
+								</dd>
+								<dt>
+									<code>
+										<var>registry</var>
```

Comment:
Why is this part of the api, does it make a difference to the build service user?

_Appreciate this PR is about docs, if you'd rather reply to this question elsewhere_

Reply:
https://github.com/Financial-Times/origami-build-service-v3/issues/32